flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/AbstractKubernetesParameters.java

```diff
@@ -159,16 +160,16 @@ public boolean hasLog4j() {
 
 	@Override
 	public Optional<String> getLocalHadoopConfigurationDirectory() {
-		final String[] possibleHadoopConfPaths = new String[] {
-			System.getenv(Constants.ENV_HADOOP_CONF_DIR),
-			System.getenv(Constants.ENV_HADOOP_HOME) + "/etc/hadoop", // hadoop 2.2
-			System.getenv(Constants.ENV_HADOOP_HOME) + "/conf"
-		};
-
-		for (String hadoopConfPath: possibleHadoopConfPaths) {
-			if (StringUtils.isNotBlank(hadoopConfPath)) {
-				return Optional.of(hadoopConfPath);
-			}
+		final String hadoopConfDirEnv = System.getenv(Constants.ENV_HADOOP_CONF_DIR);
+		final String hadoopHomeEnv = System.getenv(Constants.ENV_HADOOP_HOME);
```

Comment:
nit: We could move the variable initialization after the if block as its not needed beforehand.